pkg/qr-code-generator/Makefile

```diff
@@ -7,5 +7,10 @@ include $(RIOTBASE)/pkg/pkg.mk
 
 CFLAGS += -Wno-type-limits
 
+ifneq (,$(findstring NDEBUG,$(CFLAGS)))
+  # avoid compilation errors when NDEBUG is defined
+  CFLAGS += -Wno-unused-parameter
+endif
```

Comment:
IMO for packages we can just unconditionally disable warnings. We usually don't maintain the package code anyway, so we do not really profit from reading the warnings. Hence, there is IMO no reason to enable as many warnings as possible.

Reply:
> lgtm, comments are optional. Squash directly. I'll ACK once the testing commit is dropped.

Rebase and testing commit removed.